frontend/config/agents.ts

```diff
@@ -35,77 +40,65 @@ export const AGENT_CONFIG: {
     middlewareHomeChainId: MiddlewareChain.GNOSIS,
     requiresAgentSafesOn: [EvmChainId.Gnosis],
-    agentSafeFundingRequirements: {
-      [EvmChainId.Gnosis]: 0.1,
-    },
     operatingThresholds: {
       [WalletOwnerType.Master]: {
-        [WalletType.EOA]: {
-          [TokenSymbol.XDAI]: 0.1,
-        },
         [WalletType.Safe]: {
           [TokenSymbol.XDAI]: Number(
             formatEther(
-              `${
-                (traderFundRequirements?.[ethers.constants.AddressZero]
-                  ?.agent ?? 0) +
-                (traderFundRequirements?.[ethers.constants.AddressZero]?.safe ??
-                  0)
-              }`,
+              `${traderFundRequirements.agent + traderFundRequirements.safe}`,
             ),
           ),
         },
-      },
-      [WalletOwnerType.Agent]: {
         [WalletType.EOA]: {
-          [TokenSymbol.XDAI]: 0.1,
+          [TokenSymbol.XDAI]: 0.1, // TODO: should come from the template
         },
+      },
+      [WalletOwnerType.Agent]: {
         [WalletType.Safe]: {
-          [TokenSymbol.XDAI]: 0.1,
+          [TokenSymbol.XDAI]: Number(
+            formatEther(`${traderFundRequirements.agent}`),
+          ),
+        },
+        [WalletType.EOA]: {
+          [TokenSymbol.XDAI]: 0.1, // TODO: should come from the template
         },
       },
     },
     requiresMasterSafesOn: [EvmChainId.Gnosis],
     serviceApi: PredictTraderService,
     displayName: 'Prediction agent',
     description: 'Participates in prediction markets.',
+    eoaFunding: {
+      [EvmChainId.Gnosis]: {
+        chainConfig: CHAIN_CONFIG[EvmChainId.Gnosis],
+      },
+    },
     isAgentEnabled: true,
   },
-  // [AgentType.Optimus]: {
-  //   name: 'Optimus',
-  //   homeChainId: ChainId.Optimism,
-  //   requiresAgentSafesOn: [ChainId.Optimism, ChainId.Ethereum, ChainId.Base],
-  //   requiresMasterSafesOn: [ChainId.Optimism, ChainId.Ethereum, ChainId.Base],
-  //   agentSafeFundingRequirements: {
-  //     [ChainId.Optimism]: 100000000000000000,
-  //     [ChainId.Ethereum]: 100000000000000000,
-  //     [ChainId.Base]: 100000000000000000,
-  //   },
-  //   serviceApi: OptimusService,
-  // },
   [AgentType.Memeooorr]: {
     name: 'Agents.fun agent',
     evmHomeChainId: EvmChainId.Base,
     middlewareHomeChainId: MiddlewareChain.BASE,
     requiresAgentSafesOn: [EvmChainId.Base],
-    agentSafeFundingRequirements: {
-      [EvmChainId.Base]: 0.03,
-    },
     operatingThresholds: {
       [WalletOwnerType.Master]: {
-        [WalletType.EOA]: {
-          [TokenSymbol.ETH]: 0.0001,
-        },
         [WalletType.Safe]: {
-          [TokenSymbol.ETH]: 0.0001,
+          [TokenSymbol.ETH]: Number(
+            formatEther(`${memeooorrRequirements.safe}`),
+          ),
         },
-      },
-      [WalletOwnerType.Agent]: {
         [WalletType.EOA]: {
-          [TokenSymbol.ETH]: 0.0001,
+          [TokenSymbol.ETH]: 0.0125, // TODO: should come from the template
```

Comment:
where this value comes from? wonder why not `0.005` as it was in `MIN_ETH_BALANCE_THRESHOLDS.safeAddSigner`? however it's still unclear to me where it might be used other than for safe creation, for which we request funds anyway

Reply:
agree that we should stick to the doc - this is a product decision